src/utils/LocalDiscoveryServer.ts

```diff
@@ -68,23 +91,14 @@ export class LocalDiscoveryServer {
             const [peerPort, peerChannelId] = JSON.parse(message.toString());
             if (duplicateSet.has(peerPort)) return;
             duplicateSet.add(peerPort);
+
             if (
-                peerPort > myPort &&
+                peerPort !== myPort && // Don't connect to self
```

Comment:
@daiagi shouldn't this connect double connections? A<=>B and B<=>A 

with portA>portB only from 1 peers perspective is that relation true and a single connection is maintained - or I'm not seeing something